README.rst

```diff
@@ -8,273 +8,7 @@ pynsot
-``-o/--offset`` options.
-
-+ The ``-i/--id`` option will retrieve a single object by the provided unique
-  ID and will override any other list options.
-+ The ``-l/--limit`` option will limit the set of results to ``N`` resources.
-+ The ``-o/--offset`` option will skip the first ``N`` resources.
-
-Set Queries
-~~~~~~~~~~~
-
-The Device and Network resources support a ``-q/--query`` option that is a
-representation of set operations for matching attribute/value pairs.
-
-The operations are evaluated from left-to-right, where the first character indicates the
-set operation:
-
-+ "+" indicates a set union
-+ "-" indicates a set difference
-+ no marker indicates a set intersection
-
-For example
-
-+ ``-q "foo=bar"`` would return the set intersection of objects with ``foo=bar``.
-+ ``-q "foo=bar -owner=jathan"`` would return the set difference of all objects
-  with ``foo=bar`` (that is all ``foo=bar`` where ``owner`` is not ``jathan``).
-+ ``-q "foo=bar +foo=baz`` would return the set union of all objects with
-  ``foo=bar`` or ``foo=baz`` (that is all objects matching either).
-
-The ordering of these operations is important. If you are not familiar with set
-operations, please check out `Basic set theory concepts and notation
-<http://en.wikipedia.org/wiki/Set_theory#Basic_concepts_and_notation>`_
-(Wikipedia).
-
-Bulk Addition of Objects
-------------------------
-
-Attributes, Devices, and Networks may be created in bulk by using the
-``-b/--bulk-add`` option and specifying a file path to a colon-delimited file.
-
-The format of this file must adhere to the following format:
-
-+ The first line of the file must be the field names.
-+ All required fields must be present, however, the order of any of the fields
-  does not matter.
-+ Repeat: The fields may be in any order so long as the required fields are
-  present! Missing fields will fallback to their defaults!
-+ Attribute pairs must be commma-separated, and in format k=v and the
-  attributes must exist!
-+ For any fields that require Boolean values, the following applies:
-
-  - You may specify ``True`` or ``False`` and they will be evaluated
-  - If the value for a field is not set it will evaluate to ``False``
-  - Any other value for a field will evaluate to ``True``
-
-Attributes
-~~~~~~~~~~
-
-Sample file for ``nsot devices add --bulk-add /tmp/attributes``::
-
-    name:resource_name:required:description:multi:display
-    owner:Network:True:Network owner:True:True
-    metro:Device:False:Device metro:False:True
-
-Devices
-~~~~~~~
-
-Sample file for ``nsot devices add --bulk-add /tmp/devices``::
-
-    hostname:attributes
-    device5:foo=bar,owner=team-networking
-    device6:foo=bar,owner=team-networking
-
-Networks
-~~~~~~~~
-
-Sample file for ``nsot networks add --bulk-add /tmp/networks``::
-
-    cidr:attributes
-    10.20.30.0/24:foo=bar,owner=team-networking
-    10.20.31.0/24:foo=bar,owner=team-networking
+Read the docs at http://pynsot.rtfd.org
 
 Working with Resources
```

Comment:
First of all. This is great! Why don't we just move this entire section over, too?